### ISSUE TITLE
{REQUEST} Image utilities file saving

### ISSUE THREAD

**Issue by LuSaffi** [Vencord/plugin-requests#350]
### Plugin Description

Image utilities style image saving (pre-define a location)> Right click image > Save Image to > {FolderName}
![68747470733a2f2f6d776974747269656e2e6769746875622e696f2f426574746572446973636f72644164646f6e732f506c7567696e732f496d6167655574696c69746965732f5f7265732f73637265656e73686f74322e706e67](https://github.com/Vencord/plugin-requests/assets/58399679/279e6cb6-267c-4d32-8724-eaff987e587e)

### Existing Plugin for other mod

https://github.com/mwittrien/BetterDiscordAddons/tree/master/Plugins/ImageUtilities

### Request Agreement

- [X] I read the above and made sure my plugin is possible and allowed
- [X] I checked the [existing plugins](https://vencord.dev/plugins) and made sure my plugin doesn't already exist
- [X] I [searched the existing issues](https://github.com/Vencord/plugin-requests/issues?q=is%3Aissue) and made sure a similar request doesn't already exist

**Comment by Data-PhiIe** [comment]
> Looks like a good idea for a plugin, though some features are already in other plugins like for example zooming image (ImageZoom) or image searching (ReverseImageSearch). I might check later what can I do with this and let you know, for now let's see other's reaction.

Devilbro made one for BDD (https://betterdiscord.app/plugin/ImageUtilities). You guys could probably wrap up all of it into one plugin (zoom and reverse lookup). I made the switch to vencord this year, the user experience has been a lot better than BDD for me personally (it hasn't broken as much). The one and only plugin I feel I can't do without is the easy context menu to download images/gif/videos/emojis. For me personally, I feel its the one thing making vencord incomplete. I upvoted this request, I really hope to see it soon! :)

**Comment by LuSaffi** [comment]
Yeah i specifically would like to see all the similar functions merged into one plugin, plus the file saving context menu added, I really miss that functionality and if BD wasnt a broken mess I'd still be there _because_ of that context menu

**Comment by TheSrBig** [comment]
I love this request, I would also like to be able to download stickers, gifs, and emojis, even animated ones.

**Comment by QuestionableNinja** [comment]
I've just downloaded Vencord for the first time having never heard of it before and this plugin is vital for me. The click to zoom instead of click-and-hold, save directories are requirements for me. Having reverse image search bundled in is just the cherry on top.
